src/plugins/condition/components/ConditionEdit.vue

```diff
@@ -262,18 +301,22 @@ export default {
             }
         },
         operationKeyChange(ev) {
-            if (ev.target.value !== 'isUndefined' && ev.target.value !== 'isDefined') {
+            if (this.selectedOperationKey !== 'isUndefined' && this.selectedOperationKey !== 'isDefined') {
                 this.comparisonValueField = true;
             } else {
                 this.comparisonValueField = false;
             }
-            this.condition.definition.criteria[0].operation = this.selectedOperationKey;
-            this.persist();
+            this.updateConditionCriteria();
             //find the criterion being updated and set the operation property
         },
+        telemetryChange() {
```

Comment:
A function is an action and so the name should indicate that.

Reply:
Done